pkg/cloudprovider/aws/instance.go

```diff
@@ -140,6 +140,7 @@ func (p *InstanceProvider) launchInstances(ctx context.Context, constraints *v1a
 	tags := v1alpha1.MergeTags(ctx, constraints.Tags, map[string]string{fmt.Sprintf("kubernetes.io/cluster/%s", injection.GetOptions(ctx).ClusterName): "owned"})
 	createFleetInput := &ec2.CreateFleetInput{
 		Type:                  aws.String(ec2.FleetTypeInstant),
+		Context:               aws.String(*constraints.Context),
```

Comment:
this will actually result in a nil pointer fault.  

This should work:
```suggestion
		Context:             constraints.Context,
```